tests/test_multiline_variables.py

```diff
@@ -0,0 +1,22 @@
+from universum.lib.module_arguments import ModuleArgumentParser, multiline_argument
+
+
+def test_multiline_vars(tmp_path, monkeypatch):
+    text = "This is text\nwith some line breaks\n"
+    parser = ModuleArgumentParser()
+    parser.add_argument("--argument", type=multiline_argument)
+
+    # Usual value
+    namespace = parser.parse_args(["--argument", text])
+    assert namespace.argument == text
+
+    # File
+    tmp_file = tmp_path / "myfile.txt"
```

Comment:
I think, you can use tmpfile directly

Reply:
pytest doesn't have a 'tmp_file' fixture, only 'tmp_path' and 'tmp_dir', which are both directories. I can switch to built-in `tempfile.NamedTemporaryFile` if needed, but I myself like pytest's fixtures better, and also we do use them in all other tests.